docs/core/common/assets/js/api/core/commands.md

```diff
@@ -0,0 +1,174 @@
+## API --  `$e.commands`
+The new Commands API (since 2.7.0), provides a simple and convenient way to run something the editor, create a widget, as well as show a notice or undo changes, using JS commands.
```

Comment:
> to run something the editor

Probably you mean "to run something *on/in* the editor

Reply:
Handled.